frontend2/src/views/Rankings.tsx

```diff
@@ -64,7 +64,7 @@ const Rankings: React.FC = () => {
   }, [searchQuery, page]);
 
   return (
-    <div className="mb-20 ml-10 flex w-full flex-col">
+    <div className="flex h-full w-full flex-col overflow-auto p-6">
```

Comment:
we have to use padding instead of margins here, otherwise it will overflow the page width because of the combined `w-full` and `ml-10`